includes/Core/Util/Google_Icon.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Google_Icon
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Class for the Google SVG Icon
+ *
+ * @since n.e.x.t
```

Comment:
There are a few more annotations that we add to all classes which we should include here as well:
```suggestion
 * @since n.e.x.t
 * @access private
 * @ignore
```